experiments/phase2_distillation/step4c_prepare_mlx_train.py

```
@@ -0,0 +1,153 @@
+
+from __future__ import annotations
+
+import argparse
+import json
+import random
+import sys
+from pathlib import Path
+
+
+def _split(rows: list[dict], frac_valid: float, frac_test: float, seed: int) -> tuple[list, list, list]:
+    rng = random.Random(seed)
```

Ruff (0.15.9)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)